package.json

```diff
@@ -56,7 +56,7 @@
     "@types/compression": "^1.7.1",
     "@types/node": "18",
     "compression": "^1.7.4",
-    "jsdom": "^22.1.0",
+    "happy-dom": "^14.12.3",
```

Comment:
Aaand it looks `happy-dom` has a bug too. At least, a different behavior from jsdom: the `set-cookie` header gets stripped off when headers are set on the `Response` instance. 

```js
let h = new Headers([['set-cookie', 'a=b']])
// h.get('set-cookie') OK!

let r = new Response(null, { headers })
// r.headers.get('set-cookie') null
```

I wouldn't treat jsdom as the correct behavior here but this is breaking the tests. Even `.getSetCookie()` returns `[]`. May really be a bug in happy-dom.